src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/PresentationSource.cs

```diff
@@ -129,10 +129,10 @@ public static void AddSourceChangedHandler(IInputElement element, SourceChangedE
                 throw new ArgumentNullException("element");
             }
 
-            // Either UIElement or ContentElement
+            // Either UIElement, ContentElement or UIElement3D.
             if (!InputElement.IsValid(element))
             {
-                throw new ArgumentException(SR.Get(SRID.Invalid_IInputElement), "element");
+                throw new ArgumentException(SR.Get(SRID.Invalid_IInputElement, element.GetType()), nameof(element));
```

Comment:
Here is an example of a call to SR.Get using Invalid_IInputElement and not passing a parameter for the format.

Here's the content of Invalid_IInputElement: https://github.com/dotnet/wpf/blob/41bdb1071eb62fb04bfb57d88d94e79dedc6fdbc/src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/Strings.resx#L1289